ui/Modal/ManagePeers.svelte

```diff
@@ -83,6 +83,7 @@
     <Emoji emoji={'💻'} size="huge" style="margin-bottom: 1.5rem;" />
 
     <h1>Manage remotes</h1>
+    <p>Add a user's Device ID to collaborate with them on this project</p>
```

Comment:
```suggestion
        <p style="margin-top: 0.5rem; color: var(--color-foreground-level-6);">Add a user's Device ID to collaborate with them on this project</p>

```

Reply:
Add a period to the end of this sentence